core/rawdb/accessors_chain.go

```diff
@@ -669,15 +669,6 @@ func DeleteReceipts(db ethdb.KeyValueWriter, hash common.Hash, number uint64) {
 	}
 }
 
-// storedReceiptRLP is the storage encoding of a receipt.
-// Re-definition in core/types/receipt.go.
-// TODO: Re-use the existing definition.
-type storedReceiptRLP struct {
-	PostStateOrStatus []byte
-	CumulativeGasUsed uint64
-	Logs              []*types.Log
-}
-
```

Comment:
The entire ReadLogs function can be deprecated once https://github.com/ethereum/go-ethereum/pull/29158 is merged. Please hold on the PR a bit and remove the whole thing.